### ISSUE TITLE
Latest optimize docs links does not contain the version

### ISSUE THREAD

**Issue by OmranAbazid** [camunda/camunda-docs#2673]
### Context:
We are trying to make the docs links in Optimize, web app respect the version. 
The docs currently contain the version in the URL for all the old versions. However, the latest version does not contain the version.
There is no easy way in Optimize to figure out which version is the latest, so we do not know when to remove the version and when to keep.

### Example
in 3.9.0, the optimize doc link look like this:
`https://docs.camunda.io/optimize/3.9.0/components/what-is-optimize/`
but in 3.10.0 (latest as of writing this), appear as
`https://docs.camunda.io/optimize/components/what-is-optimize/`

### Expected outcome: 
The version always exists in the URL even for the latest version.

**Comment by OmranAbazid** [comment]
@pepopowitz Is this possible to resolve and how much effort is required?

**Comment by OmranAbazid** [comment]
There is already an .htaccess file with the redirects in it.
Can we use that file to do a redirect for the latest version?

**Comment by akeller** [comment]
Similar to https://github.com/camunda/camunda-docs/issues/1246.

**Comment by pepopowitz** [comment]
Some thoughts out loud -- 

1. I think it would be reasonable to add rewrite rules for the "current" version, as long as we specify them as 302 instead of 301. I don't want any of these rules to get stuck in someone's cache (especially not a crawler's): 

   ```
   RewriteRule ^docs/8.2/(.*)$ /docs/$1 [R=302,L]
   RewriteRule ^optimize/3.10.0/(.*)$ /optimize/$1 [R=302,L]
   ```

2. We'd have to update these rules with each versioning, but that could be scripted pretty easily, and at the very least documented clearly. 
3. I'm not worried about SEO/crawling/search issues with this. It seems like we'd only use these versioned links from within our apps, and neither users nor crawlers would see them. Do you have concerns, @akeller?

**Comment by akeller** [comment]
Within the documentation, we should never specify the version via the URL and let Docusaurus handle that when we roll versions for minor releases. So, docs authors' habits would not change.

I don't have any concerns with this @pepopowitz. 

If @OmranAbazid wants this now, I assume it's for the upcoming minor release. Would that be feasible for you @pepopowitz?  Or after this upcoming minor?

**Comment by pepopowitz** [comment]
I can take a look at this today. If it looks as straightforward as I think, before release should be no problem. If it looks troublesome, I'd push it until after the release.

**Comment by pepopowitz** [comment]
Resolved in #2679
